CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateRacing.cpp

```diff
@@ -226,7 +226,7 @@ void Kartaclysm::StateRacing::CreateHUDForRacer(const std::string& p_strGuid)
 
 void Kartaclysm::StateRacing::Suspend(const int p_iNewState)
 {
-	m_bSuspended = true;
+	m_bSuspended = (p_iNewState != STATE_COUNTDOWN);
```

Comment:
We don't want StateRacing to be suspended if we go into StateCountdown. If it's suspended, then Update() isn't called which defeats the purpose of this pull request (we want kart physics and particle timers still updating during the countdown). We still want it suspended when other states are pushed though (i.e. Pause Menu).